### ISSUE TITLE
[Pattern] Speed bumps

### ISSUE THREAD

**Issue by blinks** [blinks/pattern#50]
- [ ] Confidence, from 1 to 3 stars.
- [ ] Context (followed by `* * *`)
- [ ] Problem (in bold)
- [ ] Forces
- [ ] Solution (in bold, followed by `* * *`)
- [ ] "How and why" explanatory text.

If it feels like there's not enough game there, throw in a few extra steps / requirements / actions.

> "[Surely] I would not take a stick with a piece of metal on one end of it, walk three or four hundred yards away from the hole, and then attempt to propel the ball into the hole with the stick." (Bernard Suits, "Construction of a Definition", The Game Design Reader: A Rules of Play Anthology.)

**Comment by garykac** [comment]
"challenges"

**Comment by blinks** [comment]
Yeah, but the important part is that you're throwing something in the way of "just doing what you want" -- generally something that might seem totally unnecessary.
